src/main/java/lotto/model/LottoMachine.java

```diff
@@ -0,0 +1,145 @@
+package lotto.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import lotto.exception.ErrorMessage;
+import lotto.exception.LottoException;
+import lotto.validate.Validator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoMachine {
+    private final int MAX = 45;
+    private final int MIN = 1;
+    private final int COUNT = 6;
+    private final int MAX_PRICE = 100_000;
+    private final int PRICE = 1_000;
+
+    private List<Lotto> lottos;
+    private Lotto winner;
+    private int bonus;
```

Comment:
final 키워드를 사용해 생성자 외엔 변경이 불가능하도록 하면 더 안정적으로 사용할 수 있을 것 같아요!
```suggestion
    private List<Lotto> lottos;
    private Lotto winner;
    private int bonus;
```

Reply:
좋은 의견인것 같습니다. 감사합니다!!